internal/sdk/cloudian/sdk.go

```diff
@@ -385,15 +344,16 @@ func (client Client) GetGroup(ctx context.Context, groupId string) (*Group, erro
 	}
 }
 
-func (client Client) newRequest(ctx context.Context, method string, url string, query map[string]string, body []byte) (*http.Request, error) {
+func (client Client) newRequest(ctx context.Context, method string, url string, query map[string]string, body []byte) (*http.Response, error) {
```

Comment:
```suggestion
func (client Client) performRequest(ctx context.Context, method string, url string, query map[string]string, body []byte) (*http.Response, error) {
```

@erikgb @mariatsji please just continue on my branch until you are both satisfied

Reply:
I'm fine with just calling it `do` or maybe `doHttp` in case anyone thinks that a `client` can do multiple things..

Reply:
`doRequest`?

Reply:
```suggestion
func (client Client) doRequest(ctx context.Context, method string, url string, query map[string]string, body []byte) (*http.Response, error) {
```